.gitlab-ci.yml

```diff
@@ -201,3 +201,20 @@ clean_merged_prs:
   variables:
     DEPLOYMENT_BASE: $DEPLOYMENT_BASE
     GITHUB_PULL_REQUEST_ID: "merged"
+
+
+# some changes require a human to be notified. We'll do this through JIRA tickets
+notifications:
+  needs: [atomic_build]
+  image: python:3.11-bullseye
+  rules:
+    - if: '$CI_COMMIT_BRANCH == $DEPLOYMENT_DEFAULT_BRANCH'
```

Comment:
Sorry, I think it's not going to be _quite_ as simple as this.

If a dependency is changed, singularity would be rebuilt, too. That would require a trigger, too.

I'm wondering if we have something in the artifacts we could test for... :thinking: